integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/viewmodels/TransferDashViewModel.kt

```diff
@@ -118,7 +119,7 @@ class TransferDashViewModel @Inject constructor(
         return config.spendingConfirmationEnabled
     }
 
-    fun createAddressForAccount() = viewModelScope.launch(Dispatchers.Main){
+    fun createAddressForAccount() = viewModelScope.launch {
```

Comment:
`viewModelScope` is already launched in the main thread. Specifying it explicitly is a bit confusing.